docs/extensibility/theme-support.md

```diff
@@ -91,3 +91,63 @@ add_theme_support( 'disable-custom-colors' );
 ```
 
 This flag will make sure users are only able to choose colors from the `editor-color-palette` the theme provided or from the editor default colors if the theme did not provide one.
+
+## Editor styles
+
+A theme can provide a stylesheet to the editor itself, to change colors, fonts, and any aspect of the editor.
+
+### Add the stylesheet
+
+First thing you need to do is to enqueue the new editor style. Like this:
```

Comment:
This is missing an article and the next sentence is a fragment. It might read better as:

<pre>
The first thing to do is enqueue the new editor style:

```
/**
 * Enqueue block editor style
 */
function mytheme_block_editor_styles() {
	wp_enqueue_style( 'mytheme-block-editor-styles', get_theme_file_uri( '/style-editor.css' ), false, '1.0', 'all' );
}
add_action( 'enqueue_block_editor_assets', 'mytheme_block_editor_styles' );
```
</pre>